server.js

```diff
@@ -0,0 +1,149 @@
+// START server config
+const express = require("express")
```

Comment:
There are some code style inconsistencies with this file; try using your IDE's built-in code formatter, and if that fails, make sure there are semicolons at the end of statements and single quotes being used.